src/components/ChannelSearch/ChannelSearch.tsx

```diff
@@ -20,7 +22,7 @@ import type {
 
 export type ChannelSearchFunctionParams<Us extends DefaultUserType<Us> = DefaultUserType> = {
   setQuery: React.Dispatch<React.SetStateAction<string>>;
-  setResults: React.Dispatch<React.SetStateAction<Array<UserResponse<Us>>>>;
+  setResults: React.Dispatch<React.SetStateAction<Array<ChannelOrUserType> | []>>;
```

Comment:
do we need the empty array here?

Reply:
yes, because I'm initializing that state variable as an empty erray

Reply:
actually, no...I will take this out.